rkt/rkt.mk

```diff
@@ -29,15 +29,19 @@ $(BGB_BINARY): $(MK_PATH) | $(BINDIR)
 
 include makelib/build_go_bin.mk
 
+DIST_SRC_FILTER := '_test.go|main.go|_gen.go|_stub.go'
```

Comment:
I'd prefer if this variable was prefixed with `LOCAL_` (so, `LOCAL_DIST_SRC_FILTER`) and then add `$(call forward-vars,manpages, LOCAL_DIST_SRC_FILTER)` and `$(call forward-vars,manpages, LOCAL_DIST_SRC_FILTER)` before the relevant rule.

This will avoid spilling the `DIST_SRC_FILTER` variable to the rest of the build system (all the variables prefixed with `LOCAL_` are cleaned by the call to `undefine-namespaces` at the bottom of the file).